### PR TITLE
Minor fixes because of PHP7 strict mode

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/PhpGenerator/ClassType.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public static function from($from)
 	}
 
 
-	public function __construct($name = NULL, PhpNamespace $namespace = NULL)
+	public function __construct($name, PhpNamespace $namespace = NULL)
```
Comment (dg) [inline src/PhpGenerator/ClassType.php]: This is BC break when you use `new ClassType)->setName(...)` (used i.e by nette/di).

File: src/PhpGenerator/PhpFile.php [hunk 2]
```diff
@@ -125,7 +125,7 @@ public function addTrait($name)
 	public function addNamespace($name)
 	{
 		if (!isset($this->namespaces[$name])) {
-			$this->namespaces[$name] = new PhpNamespace($name);
+			$this->namespaces[$name] = new PhpNamespace($name ?: '');
 		}
```
Comment (dg) [inline src/PhpGenerator/PhpFile.php]: Isn't it better to add support for NULL namespace than `''`?

File: src/PhpGenerator/ClassType.php (hunk 1 above)
Comment (kukulich) [inline src/PhpGenerator/ClassType.php]: Force pushed with other solution.

File: src/PhpGenerator/PhpFile.php (hunk 2 above)
Comment (kukulich) [inline src/PhpGenerator/PhpFile.php]: Yes, however it needs more changes. Should I try to prepare it in this pull request or prepare rather another one?

File: src/PhpGenerator/ClassType.php [hunk 3]
```diff
@@ -25,7 +25,7 @@ class ClassType extends Nette\Object
 	/** @var PhpNamespace|NULL */
 	private $namespace;
 
-	/** @var string */
+	/** @var string|NULL */
 	private $name;
```
Comment (dg) [inline src/PhpGenerator/ClassType.php]: `Parameter` & `Property` uses `$name = ''` instead of `|NULL`, because `$name` is not "optional" parameter.

**Comment by kukulich** [comment]
Force pushed again.

**Comment by dg** [comment]
Great!

**Pull request merged**